tests/cpp/test_multidevice_sharding.cpp

```diff
@@ -47,7 +45,7 @@ TEST_P(MultideviceShardingTest, UnshardedGlobalInput) {
 
   tv1->axis(sharded_dim)->parallelize(ParallelType::DIDx);
   tv2->axis(sharded_dim)->parallelize(ParallelType::DIDx);
-  tv3->axis(sharded_output_dim)->parallelize(ParallelType::DIDx);
+  tv3->axis(-1)->parallelize(ParallelType::DIDx);
```

Comment:
With a concrete tensor of shape [1,...], `tv3->definition()` is made a SqueezeOp, which doesn't keep the first input dimension as `r`. Therefore, `axis(-1)` is a more reliable way to get the last dimension.